src/components/feedback-button.js

```diff
@@ -0,0 +1,38 @@
+import { css } from 'react-emotion'
+import { faCommentDots } from '@fortawesome/free-solid-svg-icons'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import React from 'react'
+import { BLUE, LIGHT_BLUE, WHITE } from '../assets/colors'
+
+export default function FeedbackButton () {
+  return (
+    <a
+      className={css`
```

Comment:
large blocks of css go variables, sort of how you wouldn't just write a huge anonymous function inside a callback, but name it somewhere else instead

Reply:
Is this a blocking comment?

Reply:
yes. I don't want to get used to this approach because it makes reading the code difficult, it takes me a few seconds to realize this is just an <a> tag (imagine if we just called huge anonymous functions everywhere instead of giving them a name).

Reply:
What would you call it? The styling is specific to this a tag which composes the feedback button

Reply:
feedbackLink, or I would even turn it completely into a styled component called FeedbackLink. Up to you